tensorflow_similarity/architectures/efficientnet.py

```diff
@@ -141,13 +144,10 @@ def EfficientNetSim(
     return SimilarityModel(inputs, outputs)
 
 
-def build_effnet(
-    x: layers.Layer, variant: str, weights: str, trainable: str
-) -> layers.Layer:
+def build_effnet(variant: str, weights: str = None, trainable: str = "full") -> tf.keras.Model:
```

Comment:
We removed the `x: layers.Layer` here, so we'll need to update line 124 to:

`x = build_effnet(variant, weights, trainable)(x)`